README.adoc

```diff
@@ -13,24 +13,27 @@ ifndef::env-github[]
 image::images/Ui.png[width="600"]
 endif::[]
 
-* This is a desktop Address Book application. It has a GUI but most of the user interactions happen using a CLI (Command Line Interface).
-* It is a Java sample application intended for students learning Software Engineering while using Java as the main programming language.
-* It is *written in OOP fashion*. It provides a *reasonably well-written* code example that is *significantly bigger* (around 6 KLoC)than what students usually write in beginner-level SE modules.
-* What's different from https://github.com/se-edu/addressbook-level3[level 3]:
-** A more sophisticated GUI that includes a list  panel and an in-built Browser.
-** More test cases, including automated GUI testing.
-** Support for _Build Automation_ using Gradle and for _Continuous Integration_ using Travis CI.
+== Introduction to MediTabs
```

Comment:
Might want to make the marketing blurb more convincing by adding some of the key features such as the warning and sorting features.

Reply:
@jtankw3 This is indeed a good suggestion. I have added some key features section which does not list all the key features but only some so as to give users a sneak peek at what our application offers.